packages/api-explorer/src/components/Header/Header.tsx

```diff
@@ -38,18 +38,26 @@ import {
 import { SpecList } from '@looker/sdk-codegen'
 import { SpecState, SpecAction } from '../../reducers'
 import { diffPath } from '../../utils'
+import { SdkLanguageSelector } from './SdkLanguageSelector'
 import { ApiSpecSelector } from './ApiSpecSelector'
 
 interface HeaderProps {
+  /** Specs to choose from */
   specs: SpecList
+  /** Current selected spec */
   spec: SpecState
+  /** Spec state setter */
   specDispatch: Dispatch<SpecAction>
+  /** Nav state setter */
   toggleNavigation: (target?: boolean) => void
   className?: string
```

Comment:
Just noticed the variable className. Unrelated to your change but what is it? Could could confusion with css class name. Consider renaming in a separate story (and documenting).

Reply:
I am not sure. Maybe @jkaster knows?

Reply:
🤷🏼  need to see how it's used in `SemanticHeader`